src/tasks/browserSync.js

```diff
@@ -14,8 +14,15 @@ class BrowserSyncTask {
 			this.options.config.server = _merge({}, this.options.config.server);
 			this.options.config.server.middleware = _merge([], this.options.config.server.middleware);
 
-			var historyApiFallbackConfig = _merge({}, this.options.historyApiFallback);
-			this.options.config.server.middleware.push(historyApiFallback(historyApiFallbackConfig));
+			let historyApiFallbackConfig;
+			if (typeof this.options.historyApiFallback === 'array') {
+				historyApiFallbackConfig = this.options.historyApiFallback;
+			} else if (typeof this.options.historyApiFallback === 'string') {
```

Comment:
Okay let me update that with Lodash as well as the Array version for consistency.